perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -1200,13 +1237,49 @@ def ListConsistencyBenchmark(storage_schema, host_to_connect=None):
                final_result)
 
   # Final clean up: delete the objects still remaining.
-  DeleteObjectsConcurrently(storage_schema,
+  DeleteObjectsConcurrently(backend,
                             per_thread_objects_written,
-                            host_to_connect,
                             per_thread_objects_deleted)
   return final_result
 
 
+def TestBackend(backend):
```

Comment:
This code is not for testing the benchmark, though. It's how I tested that I implemented the service provider backends correctly. So it does have to touch a real bucket, because I want to make sure that each backend can write to a bucket and then read its objects back.

I used it for manually testing all of the providers. I created sample buckets in the provider UIs, then did something like `object_storage_api_tests.py --storage_provider=GCS --bucket=... --scenario=TestBackend`, and if it printed the right thing, I knew the backend code was okay.